40.meta-mender/docs.md

```diff
@@ -7,6 +7,73 @@ shortcode-core:
 github: false
 ---
 
+## meta-mender (kirkstone-v2022.10)
+
+_Released: 10.27.2022_
+
+### Statistics
+
+A total of 432 lines added, 113 removed (delta 319)
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 5 (45.5%) |
+| Ole Petter Orhagen | 4 (36.4%) |
+| Lluis Campos | 2 (18.2%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Ole Petter Orhagen | 313 (68.3%) |
+| Kristian Amlie | 93 (20.3%) |
+| Lluis Campos | 52 (11.4%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Lluis Campos | 7 (6.2%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 11 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 458 (100.0%) |
+
+| Employers with the most hackers (total 3) | |
+|---|---|
+| Northern.tech | 3 (100.0%) |
+
+### Changelogs
+
+#### meta-mender (kirkstone-v2022.10)
+
+New changes in meta-mender since kirkstone-v2022.09:
+
+##### Bug Fixes
+
+* Upgrade grub-mender-grubenv to include the XFS module in the default installation.
+* Pre-3.5 mender-client recipes needs dependency on mender-artifact-info.
+* Fix build error when building image using mender-artifact
+  older than 3.10.0.
+
+##### Features
+
+* Add a bootstrap Artifact to the generated images in
+  /var/lib/mender/bootstrap.mender, so that the client can bootstrap it's database
+  without having to first run an update.
+  ([MEN-2585](https://tracker.mender.io/browse/MEN-2585))
+* Add mender-client_3.3.1 and mender-artifact_3.8.1 recipes.
+
+##### Other
+
+* Add recipe for mender-client 3.4.0
+* Add recipe for mender-artifact 3.9.0
+* Add recipe for mender-connect 2.1.0
+* Add recipe for mender-binary-delta 1.4.1
+* Add recipe for mender-monitor 1.2.1
+* Add recipe for mender-gateway 1.0.1
```

Comment:
It seems like you and me have different understanding. You have marked your recipes as features but I did as "other". Should we fix or leave it but agree for the next time?

Reply:
>     1. Patch release uses `fix:`

I don't think it is good enough. It is not "fixing" any feature, nor recipe - it is still adding something. From `meta-mender` point of view it doesn't matter if we are adding a minor or a bugfix recipe, both are "adding" things to build so I think we should treat them the same way.

With that reasoning in mind, my vote is for having them as `feat` too.

>     2. Minor/major release uses `feat:`

Yes :+1:

Reply:
Agreed f2f on first suggestion.